# This file is distributed under the same license as the Django package.
#
msgid ""
msgstr ""
"Project-Id-Version: Django\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2015-12-15 17:07+0200\n"
"PO-Revision-Date: 2010-05-13 15:35+0200\n"
"Last-Translator: Django team\n"
"Language-Team: English <en@li.org>\n"
"Language: en\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"

#: myvenv/lib/python3.4/site-packages/django/contrib/auth/admin.py:46
msgid "Personal info"
msgstr ""

#: myvenv/lib/python3.4/site-packages/django/contrib/auth/admin.py:47
msgid "Permissions"
msgstr ""

#: myvenv/lib/python3.4/site-packages/django/contrib/auth/admin.py:49
msgid "Important dates"
msgstr ""

#: myvenv/lib/python3.4/site-packages/django/contrib/auth/admin.py:129
#, python-format
msgid "%(name)s object with primary key %(key)r does not exist."
msgstr ""

#: myvenv/lib/python3.4/site-packages/django/contrib/auth/admin.py:139
msgid "Password changed successfully."
msgstr ""

#: myvenv/lib/python3.4/site-packages/django/contrib/auth/admin.py:150
#, python-format
msgid "Change password: %s"
msgstr ""

#: myvenv/lib/python3.4/site-packages/django/contrib/auth/apps.py:12
msgid "Authentication and Authorization"
msgstr ""

#: myvenv/lib/python3.4/site-packages/django/contrib/auth/forms.py:30
msgid "No password set."
msgstr ""

#: myvenv/lib/python3.4/site-packages/django/contrib/auth/forms.py:36
msgid "Invalid password format or unknown hashing algorithm."
msgstr ""

#: myvenv/lib/python3.4/site-packages/django/contrib/auth/forms.py:69
#: myvenv/lib/python3.4/site-packages/django/contrib/auth/forms.py:265
#: myvenv/lib/python3.4/site-packages/django/contrib/auth/forms.py:329
msgid "The two password fields didn't match."
msgstr ""

#: myvenv/lib/python3.4/site-packages/django/contrib/auth/forms.py:71
#: myvenv/lib/python3.4/site-packages/django/contrib/auth/forms.py:100
#: myvenv/lib/python3.4/site-packages/django/contrib/auth/forms.py:128
#: myvenv/lib/python3.4/site-packages/django/contrib/auth/forms.py:333
msgid "Password"
msgstr ""

#: myvenv/lib/python3.4/site-packages/django/contrib/auth/forms.py:73
msgid "Password confirmation"
msgstr ""

#: myvenv/lib/python3.4/site-packages/django/contrib/auth/forms.py:75
#: myvenv/lib/python3.4/site-packages/django/contrib/auth/forms.py:339
msgid "Enter the same password as above, for verification."
msgstr ""

#: myvenv/lib/python3.4/site-packages/django/contrib/auth/forms.py:101
msgid ""
"Raw passwords are not stored, so there is no way to see this user's "
"password, but you can change the password using <a href=\"password/\">this "
"form</a>."
msgstr ""

#: myvenv/lib/python3.4/site-packages/django/contrib/auth/forms.py:131
#, python-format
msgid ""
"Please enter a correct %(username)s and password. Note that both fields may "
"be case-sensitive."
msgstr ""

#: myvenv/lib/python3.4/site-packages/django/contrib/auth/forms.py:133
msgid "This account is inactive."
msgstr ""

#: myvenv/lib/python3.4/site-packages/django/contrib/auth/forms.py:196
msgid "Email"
msgstr ""

#: myvenv/lib/python3.4/site-packages/django/contrib/auth/forms.py:267
msgid "New password"
msgstr ""

#: myvenv/lib/python3.4/site-packages/django/contrib/auth/forms.py:269
msgid "New password confirmation"
msgstr ""

#: myvenv/lib/python3.4/site-packages/django/contrib/auth/forms.py:300
msgid "Your old password was entered incorrectly. Please enter it again."
msgstr ""

#: myvenv/lib/python3.4/site-packages/django/contrib/auth/forms.py:303
msgid "Old password"
msgstr ""

#: myvenv/lib/python3.4/site-packages/django/contrib/auth/forms.py:337
msgid "Password (again)"
msgstr ""

#: myvenv/lib/python3.4/site-packages/django/contrib/auth/hashers.py:251
#: myvenv/lib/python3.4/site-packages/django/contrib/auth/hashers.py:334
#: myvenv/lib/python3.4/site-packages/django/contrib/auth/hashers.py:382
#: myvenv/lib/python3.4/site-packages/django/contrib/auth/hashers.py:410
#: myvenv/lib/python3.4/site-packages/django/contrib/auth/hashers.py:443
#: myvenv/lib/python3.4/site-packages/django/contrib/auth/hashers.py:476
#: myvenv/lib/python3.4/site-packages/django/contrib/auth/hashers.py:510
msgid "algorithm"
msgstr ""

#: myvenv/lib/python3.4/site-packages/django/contrib/auth/hashers.py:252
msgid "iterations"
msgstr ""

#: myvenv/lib/python3.4/site-packages/django/contrib/auth/hashers.py:253
#: myvenv/lib/python3.4/site-packages/django/contrib/auth/hashers.py:336
#: myvenv/lib/python3.4/site-packages/django/contrib/auth/hashers.py:383
#: myvenv/lib/python3.4/site-packages/django/contrib/auth/hashers.py:411
#: myvenv/lib/python3.4/site-packages/django/contrib/auth/hashers.py:511
msgid "salt"
msgstr ""

#: myvenv/lib/python3.4/site-packages/django/contrib/auth/hashers.py:254
#: myvenv/lib/python3.4/site-packages/django/contrib/auth/hashers.py:384
#: myvenv/lib/python3.4/site-packages/django/contrib/auth/hashers.py:412
#: myvenv/lib/python3.4/site-packages/django/contrib/auth/hashers.py:444
#: myvenv/lib/python3.4/site-packages/django/contrib/auth/hashers.py:477
#: myvenv/lib/python3.4/site-packages/django/contrib/auth/hashers.py:512
msgid "hash"
msgstr ""

#: myvenv/lib/python3.4/site-packages/django/contrib/auth/hashers.py:335
msgid "work factor"
msgstr ""

#: myvenv/lib/python3.4/site-packages/django/contrib/auth/hashers.py:337
msgid "checksum"
msgstr ""

#: myvenv/lib/python3.4/site-packages/django/contrib/auth/models.py:64
#: myvenv/lib/python3.4/site-packages/django/contrib/auth/models.py:115
msgid "name"
msgstr ""

#: myvenv/lib/python3.4/site-packages/django/contrib/auth/models.py:66
msgid "codename"
msgstr ""

#: myvenv/lib/python3.4/site-packages/django/contrib/auth/models.py:70
msgid "permission"
msgstr ""

#: myvenv/lib/python3.4/site-packages/django/contrib/auth/models.py:71
#: myvenv/lib/python3.4/site-packages/django/contrib/auth/models.py:117
msgid "permissions"
msgstr ""

#: myvenv/lib/python3.4/site-packages/django/contrib/auth/models.py:122
msgid "group"
msgstr ""

#: myvenv/lib/python3.4/site-packages/django/contrib/auth/models.py:123
#: myvenv/lib/python3.4/site-packages/django/contrib/auth/models.py:311
msgid "groups"
msgstr ""

#: myvenv/lib/python3.4/site-packages/django/contrib/auth/models.py:196
msgid "password"
msgstr ""

#: myvenv/lib/python3.4/site-packages/django/contrib/auth/models.py:197
msgid "last login"
msgstr ""

#: myvenv/lib/python3.4/site-packages/django/contrib/auth/models.py:308
msgid "superuser status"
msgstr ""

#: myvenv/lib/python3.4/site-packages/django/contrib/auth/models.py:309
msgid ""
"Designates that this user has all permissions without explicitly assigning "
"them."
msgstr ""

#: myvenv/lib/python3.4/site-packages/django/contrib/auth/models.py:312
msgid ""
"The groups this user belongs to. A user will get all permissions granted to "
"each of their groups."
msgstr ""

#: myvenv/lib/python3.4/site-packages/django/contrib/auth/models.py:317
msgid "user permissions"
msgstr ""

#: myvenv/lib/python3.4/site-packages/django/contrib/auth/models.py:318
msgid "Specific permissions for this user."
msgstr ""

#: myvenv/lib/python3.4/site-packages/django/contrib/auth/models.py:385
msgid "username"
msgstr ""

#: myvenv/lib/python3.4/site-packages/django/contrib/auth/models.py:386
msgid "Required. 30 characters or fewer. Letters, digits and @/./+/-/_ only."
msgstr ""

#: myvenv/lib/python3.4/site-packages/django/contrib/auth/models.py:390
msgid ""
"Enter a valid username. This value may contain only letters, numbers and @/./"
"+/-/_ characters."
msgstr ""

#: myvenv/lib/python3.4/site-packages/django/contrib/auth/models.py:395
msgid "A user with that username already exists."
msgstr ""

#: myvenv/lib/python3.4/site-packages/django/contrib/auth/models.py:397
msgid "first name"
msgstr ""

#: myvenv/lib/python3.4/site-packages/django/contrib/auth/models.py:398
msgid "last name"
msgstr ""

#: myvenv/lib/python3.4/site-packages/django/contrib/auth/models.py:399
msgid "email address"
msgstr ""

#: myvenv/lib/python3.4/site-packages/django/contrib/auth/models.py:400
msgid "staff status"
msgstr ""

#: myvenv/lib/python3.4/site-packages/django/contrib/auth/models.py:401
msgid "Designates whether the user can log into this admin site."
msgstr ""

#: myvenv/lib/python3.4/site-packages/django/contrib/auth/models.py:403
msgid "active"
msgstr ""

#: myvenv/lib/python3.4/site-packages/django/contrib/auth/models.py:404
msgid ""
"Designates whether this user should be treated as active. Unselect this "
"instead of deleting accounts."
msgstr ""

#: myvenv/lib/python3.4/site-packages/django/contrib/auth/models.py:406
msgid "date joined"
msgstr ""

#: myvenv/lib/python3.4/site-packages/django/contrib/auth/models.py:414
msgid "user"
msgstr ""

#: myvenv/lib/python3.4/site-packages/django/contrib/auth/models.py:415
msgid "users"
msgstr ""

#: myvenv/lib/python3.4/site-packages/django/contrib/auth/templates/registration/password_reset_subject.txt:2
#, python-format
msgid "Password reset on %(site_name)s"
msgstr ""

#: myvenv/lib/python3.4/site-packages/django/contrib/auth/views.py:102
msgid "Logged out"
msgstr ""

#: myvenv/lib/python3.4/site-packages/django/contrib/auth/views.py:188
msgid "Password reset"
msgstr ""

#: myvenv/lib/python3.4/site-packages/django/contrib/auth/views.py:203
msgid "Password reset sent"
msgstr ""

#: myvenv/lib/python3.4/site-packages/django/contrib/auth/views.py:242
msgid "Enter new password"
msgstr ""

#: myvenv/lib/python3.4/site-packages/django/contrib/auth/views.py:253
msgid "Password reset unsuccessful"
msgstr ""

#: myvenv/lib/python3.4/site-packages/django/contrib/auth/views.py:273
msgid "Password reset complete"
msgstr ""

#: myvenv/lib/python3.4/site-packages/django/contrib/auth/views.py:310
msgid "Password change"
msgstr ""

#: myvenv/lib/python3.4/site-packages/django/contrib/auth/views.py:326
msgid "Password change successful"
msgstr ""
